maat-court-data-api/src/main/java/gov/uk/courtdata/dces/request/UpdateConcoreContributionStatusRequest.java

```diff
@@ -0,0 +1,16 @@
+package gov.uk.courtdata.dces.request;
+
+import gov.uk.courtdata.enums.ConcorContributionStatus;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class UpdateConcoreContributionStatusRequest {
```

Comment:
Sorry, last bit of feedback, I think there is a typo in the class name, it should probably be
`UpdateConcorContributionStatusRequest` without the extra `e`.